libs/frontend/ui-widgets/src/lib/InfoBubble.vue

```diff
@@ -73,20 +73,38 @@ const props = withDefaults(
 )
 const { hideDelay, showArrow, placement } = toRefs(props)
 
-const { itemRef, arrowStyle, tooltipRef, tooltipStyle } = useTooltip({
+const {
+  itemRef,
+  arrowStyle,
+  tooltipRef,
+  tooltipStyle,
+  show,
+  tooltipMouseEnter,
+  tooltipMouseLeave,
+} = useTooltip({
   placement: placement.value,
   arrow: showArrow.value,
   offset: 8,
   shift: true,
 })
-const show = ref(false)
 let hoverTimer: ReturnType<typeof setTimeout> | undefined
 
 const clickInfo = () => {
-  show.value = !show.value
+  setShow(!show.value)
   cancelBubbleDelay()
 }
 
+const setShow = (newShow: boolean) => {
```

Comment:
Rewrote all consumers of `useTooltip` to use `show` provided by the composable, instead of trying to track separately. `tooltipMouseEnter` should probably be renamed, since it's now used generally to indicate the tooltip should be shown.